cmd/tag-show.go

```diff
@@ -0,0 +1,186 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	tagging "github.com/minio/minio/pkg/bucket/object/tagging"
+	"github.com/minio/minio/pkg/console"
+)
+
+var tagShowCmd = cli.Command{
+	Name:   "show",
+	Usage:  "show tags for objects",
+	Action: mainShowTag,
+	Before: setGlobalsFromContext,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `Name:
+	{{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} [COMMAND FLAGS] TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+DESCRIPTION:
+   Show tags assigned to an object.
+
+EXAMPLES:
+  1. Show the tags assigned to the object named testobject in the bucket testbucket on alias s3.
+     {{.Prompt}} {{.HelpName}} s3/testbucket/testobject
+
+`,
+}
+
+// tagMessage container for displaying tag
+type tagMessage struct {
+	AliasName string `json:"aliasName"`
```

Comment:
Why do you need to display `alias`, `host`

Reply:
I will remove it.